spring-web/src/main/java/org/springframework/http/client/reactive/ReactorClientHttpRequest.java

```diff
@@ -45,6 +48,8 @@
  */
 class ReactorClientHttpRequest extends AbstractClientHttpRequest implements ZeroCopyHttpOutputMessage {
 
+	public static final String ATTRIBUTES_CHANNEL_KEY = "attributes";
```

Comment:
The attribute name should be a little more qualified. Typically we would use something like: 
```java
ReactorClientHttpRequest.class.getName() + ".attributes"
```